docs/user_manual/data-model.md

```diff
@@ -54,7 +54,7 @@ together. An `appliance` is a component that is connected (coupled) to a node, a
 
 The figure below shows a simple example:
 
-```
+```txt
```

Comment:
Seems to be not consistent. Here it adds txt only to the start of the text, in other file also at the end

Reply:
Which one specifically? that would not be correct. The correct format should be 3 backticks + the filetype identifier as a content begin, and only 3 backticks as a content end

Reply:
Think I misread somewhere - can't find the other